app/models/search.rb

```diff
@@ -0,0 +1,11 @@
+class Search < ActiveRecord::Base
+  belongs_to :user
+
+  def results
+    Development.periscope(query)
+  end
+
+  def unsaved?
```

Comment:
Interesting - just out of curiosity, why is `saved?` not sufficient? Feels a little [Newspeak-y](https://en.wikipedia.org/wiki/Newspeak#Prefixes). Otherwise, it's a goodthink!